### PR TITLE
#168981040 api endpoint for updating / editing the article using database

### PR DESCRIPTION
**Pull request by mutabazigakuba**
#### What does this PR do?
it is an endpoint that helps the user to update the content of the article into the system 

#### Description of the tasks to be completed
- Adding the update the article controller method 
- adding the route for the endpoint

#### How should this be manually tested?
 - while running the server locally and postman open, add the link below
`localhost:3000/api/v2/article/:articleid` with `PATCH` method

#### Any background context to provide?
N/A

#### What are the relevant pivotal tracker stories?
visit the [Pivotal Tracker](https://www.pivotaltracker.com/n/projects/2396192)

#### Screenshots if any
Screen Shot 2019-10-09 at 10 08 58

### CODE REVIEW

File: server/controllers/userControllerv2.js [hunk 1]
```diff
@@ -42,6 +50,45 @@ const UserControllerv2 = {
             })
         }
     },
+
```
Comment (Mirriam-Maina) [inline server/controllers/userControllerv2.js]: @mutabazigakuba why is the login controller on this PR?

File: server/middleware/authv2.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import jwt from 'jsonwebtoken';
```
Comment (Mirriam-Maina) [inline server/middleware/authv2.js]: @mutabazigakuba This should not be on this PR

File: server/controllers/articlesControllerv2.js [hunk 3]
```diff
@@ -0,0 +1,61 @@
+import '@babel/polyfill';
+import db from '../config/savedb';
+import Helper from '../helpers/helper';
+
+const ArticleControllerv2 = {
+    async createArticle(req, res) {
```
Comment (Mirriam-Maina) [inline server/controllers/articlesControllerv2.js]: @mutabazigakuba This is create an article, it should not be on this PR

File: server/controllers/userControllerv2.js (hunk 1 above)
Comment (mutabazigakuba) [reply server/controllers/userControllerv2.js]: > @mutabazigakuba why is the login controller on this PR?

am always building from the previous branch e.g if i create B from A, then C is created off branch B.

Comment (Mirriam-Maina) [reply server/controllers/userControllerv2.js]: Rebase the changes from that branch. The current branch should only have the changes made on top of the branch you branched from;

steps
- Checkout to that initial branch you branched from
- do: `git pull`
- checkout to the branch with this PR
- do: `git pull --rebase origin original-branch-name`
- you might experience some merge conflicts which you will resolve
- once all merge conflicts are fixed,
- do: `git rebase --continue`
- do: `git push`

Comment (mutabazigakuba) [reply server/controllers/userControllerv2.js]: okay, thanks.

**Pull request merged**